services/github-service-api/src/main/java/org/kontinuity/catapult/service/github/api/GitHubService.java

```diff
@@ -20,8 +26,7 @@
      * @throws NoSuchRepositoryException If the specified repository does not exist
      * @throws IllegalArgumentException  If the repository name is not specified
      */
-    GitHubRepository fork(String repositoryFullName) throws NoSuchRepositoryException,
-            IllegalArgumentException;
+    GitHubRepository fork(String repositoryFullName);
```

Comment:
How is removing explicit throws of RuntimeExceptions more helpful?  IMO much less clear as to some of the conditions than my occur based on bad input.